db/schema.rb

```diff
@@ -878,6 +878,15 @@
     t.index ["website_url"], name: "index_podcast_episodes_on_website_url"
   end
 
+  create_table "podcast_ownerships", force: :cascade do |t|
+    t.datetime "created_at", precision: 6, null: false
+    t.bigint "podcast_id"
+    t.datetime "updated_at", precision: 6, null: false
+    t.bigint "user_id"
+    t.index ["podcast_id"], name: "index_podcast_ownerships_on_podcast_id", unique: true
+    t.index ["user_id"], name: "index_podcast_ownerships_on_user_id", unique: true
```

Comment:
I don't this is correct @Manasa2850, it doesn't reflect what's written in `db/migrate/20201114130315_create_podcast_ownerships.rb`

If you run the revert the migration and re-run it you'll notice the following changes from this version:

```diff
diff --git a/db/schema.rb b/db/schema.rb
index 526190a79..00ddfdfa7 100644
--- a/db/schema.rb
+++ b/db/schema.rb
@@ -880,11 +880,12 @@
 
   create_table "podcast_ownerships", force: :cascade do |t|
     t.datetime "created_at", precision: 6, null: false
-    t.bigint "podcast_id"
+    t.bigint "podcast_id", null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.bigint "user_id"
-    t.index ["podcast_id"], name: "index_podcast_ownerships_on_podcast_id", unique: true
-    t.index ["user_id"], name: "index_podcast_ownerships_on_user_id", unique: true
+    t.bigint "user_id", null: false
+    t.index ["podcast_id", "user_id"], name: "index_podcast_ownerships_on_podcast_id_and_user_id", unique: true
+    t.index ["podcast_id"], name: "index_podcast_ownerships_on_podcast_id"
+    t.index ["user_id"], name: "index_podcast_ownerships_on_user_id"
   end
```

A tip: now that we have a unique index on both columns, we don't need the individual indexes on each column, see `index: true` in the migation file. 

Could you address that?

Basically you need to run:

- `bundle exec rails db:rollback` or `rails db:migrate:down VERSION=20201114130315`
- remove the `index: true` from both columns in the migration file
- re-run `bundle exec rails db:migrate`
- check the changes to the schema and commit those that pertain to `podcast_ownerships`
- push the code

Let me know if anything isn't clear

Reply:
@rhymes I ran the migration again!

Reply:
> A tip: now that we have a unique index on both columns, we don't need the individual indexes on each column, see index: true in the migation file.

Can you also remove the `index: true` from the singular fields? See my instructions above

Reply:
@Manasa2850 don't forget to re-run the migration after having removed `index: true`

Reply:
@rhymes yup I've done that!

Reply:
These two lines tell me that the migration still contain separate indexes for those columns:

```ruby
    t.index ["podcast_id"], name: "index_podcast_ownerships_on_podcast_id"
    t.index ["user_id"], name: "index_podcast_ownerships_on_user_id"
```

If you check the documentation for [references/add_reference](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference) you'll notice how `true` is the default for the `index` param. I guess it needs to be set as `false`. Could you address that?

Reply:
@rhymes running `rails db:migrate` was still not reflecting the change :(
So I had to remove those 2 lines manually

Reply:
@Manasa2850 that's because you need to set explicitly `index: false` in the migration, see my comment above. The migration file and the schema need to reflect each other, they can't be misaligned :)